src/main/java/gov/nih/nci/evs/api/service/SparqlQueryManagerServiceImpl.java

```diff
@@ -569,6 +577,7 @@ public List<Concept> getConcepts(
     try {
       executor.awaitTermination(10, TimeUnit.MINUTES);
     } catch (final InterruptedException e) {
+      auditError(e, "getConcepts");
```

Comment:
This is one of those situations - just remove the try complete.  it's not doing anything.  The error will bubble up.